stable/distribution/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # JFrog Distribution Chart Changelog
 All changes to this project chart be documented in this file.
 
+## [2.1.0] - Jan 13, 2019
+* Update Distribution version 1.5.0
+* Remove MongoDB completely. This means an upgrade to this version must go through version 1.4.0! 
```

Comment:
add reference to relevant chart version 2.0.0

Reply:
> remove mongodb related sections in distribution-statefulset.yaml and _helpers.tpl

> add reference to relevant chart version 2.0.0

Done